app/src/ui/repository.tsx

```diff
@@ -400,6 +400,11 @@ export class RepositoryView extends React.Component<
     const selectedCommit =
       sha != null ? this.props.state.commitLookup.get(sha) || null : null
 
+    const isLocal =
+      selectedCommit == null
+        ? false
+        : this.props.state.localCommitSHAs.includes(selectedCommit.sha)
```

Comment:
I personally find his a bit cleaner and less confusing:

```typescript
const isLocal = selectedCommit != null && this.props.state.localCommitSHAs.includes(selectedCommit.sha)
```

(beware of formatting if you copy&paste it, or our CI will complain 😅 )

Reply:
Fixed in 2feae3e1447fd2bc0d18e4838e17b769a66a7b5f